### ISSUE TITLE
2019 Visual Studio Community: error MSB4019 msbuild help

### ISSUE THREAD

**Issue by stephen147** [brookhong/KeyCastOW#15]
I'm getting an error code: `error MSB4019`. Google machine didn't help much.

I've just installed VS and everything is up to date. Any help would be appreciated.

```
"C:\Users\user\Desktop\KeyCastOW-master\keycastow.vcxproj" (default target) (1) ->
  C:\Users\user\Desktop\KeyCastOW-master\keycastow.vcxproj(34,3): error MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.22
```

**Comment by stephen147** [comment]
I've fixed the issue help. See here:
https://stackoverflow.com/a/56726874/8262102

**TLDR**
Open the `keycastow.vcxproj` with notepad, change here the `<PlatformToolset>v120</PlatformToolset>` to `<PlatformToolset>v142</PlatformToolset>` to use the VC++ 2019 platform or [change the version via UI][1]



    

> In Visual Studio, in Solution Explorer, open the shortcut menu for
> your project (not for your solution) and then choose Properties to
> open your project Property Pages dialog box.
> 
>  1.   In the Property Pages dialog box, open the Configuration drop-down list and then select All Configurations.
> 
>  2.   In the left pane of the dialog box, expand Configuration Properties and then select General.
> 
>  3.   In the right pane, select Platform Toolset and then select the toolset you want from the drop-down list **(2019 - v142 in your case)**
> 
>  4.   Choose the OK button.

Now you compile the project without installing anything.
 
  [1]: https://docs.microsoft.com/en-US/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=vs-2019#to-change-the-project-toolset
